Documentation/helm-values.rst

```diff
@@ -838,7 +842,7 @@
      - object
      - ``{}``
    * - monitor
-     - Specify the CIDR for native routing (ie to avoid IP masquerade for). This value corresponds to the configured cluster-cidr. nativeRoutingCIDR:
+     - Specify the IPv4 CIDR for native routing (ie to avoid IP masquerade for). This value corresponds to the configured cluster-cidr. ipv4NativeRoutingCIDR:
```

Comment:
The placement of this option seems a bit odd like it's in the middle of the documentation for the `monitor` option , but that's not a new problem introduced by this PR.

Reply:
Right, this should probably be addressed independently. The file is generated from the comments in install/kubernetes/cilium/values.yaml:

```yaml
# -- Specify the IPv4 CIDR for native routing (ie to avoid IP masquerade for).
# This value corresponds to the configured cluster-cidr.
# ipv4NativeRoutingCIDR:
```

@bmcustodio do you know if there is a way to avoid appending the name of the variable from this comment to the generated reference? Do you think removing the space between the `#` and the variable name would work?

Apologies for the python binary, that was obviously a mistake that I did not notice. I should probably add it to a .gitignore, I think it's generated from the custom filter for spelling I added for WireGuard (but I need to check first that it's still used and not some old artefact from a development version of that filter).

Reply:
Now updated to remove the binary.